nemo/core/classes/common.py

```diff
@@ -81,16 +82,21 @@ def _attach_and_validate_output_types(self, out_objects):
                     res.neural_type = out_types_list[ind][1]
 
 
-class Serialization(ABC):
+class Serialization:
     @classmethod
     def from_config_dict(cls, config: DictConfig):
         """Instantiates object using DictConfig-based configuration"""
+
         if 'cls' in config and 'params' in config:
-            # regular hydra-based instantiation
-            instance = hydra.utils.instantiate(config=config)
+            class_obj = eval(config.cls)
+            instance = class_obj(cfg=config)
```

Comment:
Please note that this solution currently work only for Classes that have "cls" in their __init__.

I have few ideas here - the question is how we generally envision serialization/deserialization. We need to sync on this.

Reply:
OK, will do outside this PR